package pl.kawaleria.auctsys.auctions.dto.exceptions

class AuctionNotFoundException : RuntimeException() {
}